server/db/dbcontrollers/stocksController.js

```diff
@@ -25,5 +25,53 @@ module.exports = function (knex) {
       .orWhere(knex.raw('UPPER(name) like UPPER(?)', [searchLike]));
   };
 
+  //pass in array of objects contaning symbol,shares,and price 
+  //query stock_prices and update price if needed and return
+  module.getPrices = function (stockArr) {
+    
+    return knex.select('symbol','ask')
+      .from('stock_prices')
+    .then(function (dataArr) {
```

Comment:
this is going to return every stock in the stock_prices table every time this call gets made, right? (to the controller, not back to the UI that's asking for new prices).

That sounds like something that would be expensive to do as frequently as this call is going to be made. Would it be possible to pull the symbols out of the stockArr that's passed in and use those in a where on the initial query? That would cut the initial request from the DB down from 6000 stocks to a max of 25.